File: day003/Day003.cs
using System;

	class Day003
{
	static void Main()
	{
	int x, y;
	Console.Write("Dikdortgenin kisa kenarini giriniz: ");
	x = Convert.ToInt32(Console.ReadLine());
	Console.Write("Dikdortgenin uzun kenarini giriniz: ");
	y = Convert.ToInt32(Console.ReadLine());
	Console.WriteLine("Olculerini girdiginiz dikdortgenin alani: " + (x*y)); 
	}
}

File: day003/day003.c
#include <stdio.h>

int main() {
	int x, y;
	printf("Dikdortgenin uzun kenarini giriniz: ");
	scanf("%d", &x);
	printf("Dikdortgenin kisa kenarini giriniz: ");
	scanf("%d", &y);
	printf("Olculerini girdiginiz dikdortgenin alani: %d", (x*y));
	return 0;
}
